### ISSUE TITLE
Blender2Godot

### ISSUE THREAD

**Issue by ShalokShalom** [godotengine/godot-blender-exporter#373]
How does it make sense, to cooperate with this one?
https://m.youtube.com/watch?v=VNiu_bieXLw

**Comment by Calinou** [comment]
I think it's better for that add-on to remain third-party. We don't necessarily have the manpower to maintain a "kitchen sink" add-on that does a lot of different things. We would rather focus on maintaining an add-on that does *just* scene exporting.

**Comment by ShalokShalom** [comment]
Yeah, I mean we currently effectively maintain that scene exporting quite a couple of times, when you take this add-on into account. And idk how gltf is involved in that, while this seems all a bit split.
